common/kafka/kafka_consumer.h

```diff
@@ -15,34 +15,32 @@
 #pragma once
 
 #include <atomic>
-#include <map>
-#include <memory>
 #include <string>
 #include <unordered_set>
 
-#include "gflags/gflags.h"
-#include "glog/logging.h"
-#include "librdkafka/rdkafkacpp.h"
+#include "common/MultiFilePoller.h"
+#include "common/kafka/kafka_consumer_holder.h"
+
 
 namespace kafka {
 
 class KafkaConsumer {
 public:
-  KafkaConsumer(const std::unordered_set<uint32_t>& partition_ids,
-                const std::string& broker_list,
-                const std::unordered_set<std::string>& topic_names,
-                const std::string& group_id,
-                const std::string& kafka_consumer_type);
+  KafkaConsumer(const std::unordered_set<uint32_t> &partition_ids,
```

Comment:
In rocksplicator repo, we have been using following space style, i.e., const T& t;

Can you undo these changes.

Reply:
sure. np.